docker-entrypoint.sh

```diff
@@ -29,16 +29,21 @@ if [ "$1" = 'postgres' ]; then
 		fi
 		
 		: ${POSTGRES_USER:=postgres}
-		if [ "$POSTGRES_USER" = 'postgres' ]; then
-			op='ALTER'
-		else
-			op='CREATE'
+		: ${POSTGRES_DB:=$POSTGRES_USER}
```

Comment:
I'm also just now realizing this is a break from the current behavior, where specifying `POSTGRES_USER` still gives me a default `POSTGRES_DB` of `postgres` -- I'm not sure which way to feel.  I think I prefer this behavior, personally, especially since it's more consistent with what I think the user expectations would be.